docs/Manpage.md

```diff
@@ -1853,6 +1853,8 @@ This sanitized build environment ignores unrequested dependencies, which makes s
   `list` Mac App Store dependencies.
 * `--whalebrew`:
   `list` Whalebrew dependencies.
+* `--vscode`:
```

Comment:
What does the vscode flag have to do with `maintainers.json`?

Reply:
@philoserf Please don't leave comments like this on PRs, thanks.